### ISSUE TITLE
FR: Convert "floating-label" span tag to label tag

### ISSUE THREAD

**Issue by flowmars** [asuite/auth0-deploy#7]
In order to enhance accessibility, the span should be turned into a label tag.

**E.g.**
`<span class="floating-label">Email Address</span>`
-->
`<label class="floating-label" for="login-email">Email Address</label>`

**Additional considerations**
* Changing label class to either (1) no class attached or (2) changing the class to "float"

*Option 1 is chosen since current structure is similar (no classes used for forms and inputs).*
